localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,390 @@
+ queen_mada_female_blood_troll_king:0 "Queen Ma'da"
+ divine_mada_female_blood_troll_emperor:0 "Divine Ma'da" 
  
- lich_king_male:0 "Lich King"
- lich_king_female:0 "Lich Queen"
+ ### Drakkari ###
+ # Feudal
+ frost_king_feudal_king:0 "Frost King"
+ frost_queen_feudal_king:0 "Frost Queen"
+
+ ### Dragons ###
+ # Feudal
+ overlord_male_dragon_duke:0 "Overlord"
+ overlady_female_dragon_duke:0 "Overlady"
+ lord_male_dragon_king:0 "Lord"
+ lady_female_dragon_king:0 "Lady"
+ dragonlord_male_dragon_emperor:0 "Dragon Lord"
+ dragonlady_female_dragon_emperor:0 "Dragon Lady"
+
+ prince_male_dragon_kingdom:0 "Prince"
+ princess_female_dragon_kingdom:0 "Princess"
+ prince_male_dragon_empire:0 "Prince"
+ princess_female_dragon_empire:0 "Princess"
+
+ ### Red Dragons ###
+ dragonprince_male_red_dragon_king:0 "Dragon Prince"
+ dragonprincess_female_red_dragon_king:0 "Dragon Princess"
+ dragonking_male_red_dragon_emperor:0 "Dragon King"
+ dragonqueen_female_red_dragon_emperor:0 "Dragon Queen"
+
+ ### Mantids ###
+ emperor_male_mantid:0 "Grand Emperor"
+ empress_female_mantid:0 "Grand Empress"
+
+ imperial_vizier_male_mantid:0 "Imperial Vizier"
+ imperial_vizier_female_mantid:0 "Imperial Vizier"
+ grand_vizier_male_mantid:0 "Grand Vizier"
+ grand_vizier_female_mantid:0 "Grand Vizier"
+ vizier_male_mantid:0 "Vizier"
+ vizier_female_mantid:0 "Vizier"
+
+ ### Qiraji ###
+ # Special
+ emperor_ahn_qiraj_male_qiraji:0 "Emperor"
+ empress_ahn_qiraj_female_qiraji:0 "Emperor"
+ empire_ahn_qiraj:0 "Empire"
+
+ # Others
+ lord_male_qiraji_count:0 "Lord"
+ lady_female_qiraji_count:0 "Lady"
+ hive_qiraji_county:0 "Hive"
+ high_overlord_male_qiraji_duke:0 "High Overlord"
+ high_overlady_female_qiraji_duke:0 "High Overlady"
+ brood_qiraji_duchy:0 "Brood"
+
+ ### Wildhammer ###
+ # Feudal
+ chief_male_wildhammer_count:0 "Chief"
+ chieftess_female_wildhammer_count:0 "Chieftess"
+ chiefdom_wildhammer_county:0 "Chiefdom"
+ thane_male_wildhammer_duke:0 "Thane"
+ thane_female_wildhammer_duke:0 "Thane"
+ high_chiefdom_wildhammer_duchy:0 "High Chiefdom"
+ high_thane_male_wildhammer_king:0 "High Thane"
+ high_thane_female_wildhammer_king:0 "High Thane"
+
+ ### Dark Iron ###
+ # Feudal
+ lord_male_darkiron_count:0 "Lord"
+ lady_female_darkiron_count:1 "Lady"
+ lordship_darkiron_county:0 "Lordship"
+ thane_male_darkiron_duke:0 "Thane"
+ thane_female_darkiron_duke:0 "Thane"
+
+ ### Old Dwarven / Frostborn ###
+ # Feudal
+ chief_male_old_dwarven_count:0 "Chief"
+ chieftess_female_old_dwarven_count:0 "Chieftess"
+ chiefdom_old_dwarven_county:0 "Chiefdom"
+ thane_male_old_dwarven_duke:0 "Thane"
+ thane_female_old_dwarven_duke:0 "Thane"
+ high_chiefdom_old_dwarven_king:0 "High Chiefdom"
+ high_thane_male_old_dwarven_king:0 "High Thane"
+ high_thane_female_old_dwarven_kingdom:0 "High Thane"
+
+ ### Vrykul and Kvaldir ###
+ # Feudal
+ thane_male_vrykul_count:0 "Thane"
+ thane_female_vrykul_count:0 "Thane"
+ chiefdom_vrykul_county:0 "Chiefdom"
```

Comment:
Why not call it Thanedom? because a Kingdom translate to Domain of the King/King's domain. I don't think Thanedom sounds weird.

Reply:
done in 0f5ac0a93